app/services/location_service.py

```
@@ -99,4 +103,11 @@ def check_surge_pricing(
                 booking_count += 1
         
         # Apply surge if more than 5 bookings in the area
-        return booking_count > 5 
\ No newline at end of file
+        fare_info = FareCalculator.calculate_fare(
```

Ruff (0.11.9)

Local variable `fare_info` is assigned to but never used

Remove assignment to unused variable `fare_info`

(F841)